src/Tizen.NUI/src/public/WebView/WebContext.cs

```diff
@@ -213,7 +213,7 @@ public bool CacheEnabled
         /// <summary>
         /// App id.
```

Comment:
in text: id -> ID

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520